broker-plugin/plugin/src/main/sh/init-broker.sh

```diff
@@ -8,6 +8,8 @@ BROKER_PLUGIN_DIR=/opt/broker-plugin
 BROKER_CUSTOM=${BROKER_DIR}/custom
 CONFIG_TEMPLATES=${BROKER_PLUGIN_DIR}/conf
 BROKER_CONF_DIR=${BROKER_CUSTOM}/conf
+MESSAGE_EXPIRY_SCAN_PERIOD=${MESSAGE_EXPIRY_SCAN_PERIOD:-30000}
```

Comment:
Should we expose it in infra config?

Reply:
@lulf I'm not sure, I don't think most customers would want to tune this and I would actively discourage business use-cases that relied on timely TTL (that's not its intended purpose).  I'd be happy to add it.

Reply:
I think its ok to leave it out for now.